microsoft-edge/webview2/reference/javascript/webviewmessageevent.yml

```
@@ -0,0 +1,44 @@
+### YamlMime:TSType
+name: WebViewMessageEvent
+uid: 'WebView2Script!WebViewMessageEvent:class'
+package: WebView2Script!
+fullName: WebViewMessageEvent
+summary: ''
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: interface
+properties:
+  - name: additionalObjects
+    uid: 'WebView2Script!WebViewMessageEvent#additionalObjects:member'
+    package: WebView2Script!
+    fullName: additionalObjects
+    summary: >-
+      Additional DOM objects that are sent via PostJSONMessageWithAdditionalObjects. These objects can be of following
+      types:
+      
+      DOM | Win32
+      ------- | ------------
+      [FileSystemFileHandle](https://developer.mozilla.org/docs/Web/API/FileSystemFileHandle) | ICoreWebView2FileSystemHandle (kind File)
+      [FileSystemDirectoryHandle](https://developer.mozilla.org/docs/Web/API/FileSystemDirectoryHandle) ICoreWebView2FileSystemHandle (kind Directory)

```

```suggestion
      [FileSystemDirectoryHandle](https://developer.mozilla.org/docs/Web/API/FileSystemDirectoryHandle) ICoreWebView2FileSystemHandle (kind `Directory`)
```
if backticks are supported here & arguably appropriate.
suggest some such formatting hints to differentiate English regular words vs. code key words.

---

what's the rendered URL?  need to see rendered, to recommend formatting